web/screenshotTesting/conf.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter, StacktraceOption } = require('jasmine-spec-reporter');
+const { join } = require('path');
+
+/**
+ * @type { import("protractor").Config }
+ */
+exports.config = {
+  allScriptsTimeout: 11000,
+  specs: [
+    './*.screenshot-spec.ts'
+  ],
+  capabilities: {
+    browserName: 'chrome',
+    chromeOptions: {
+      args: ["--headless"]
+    }
+  },
+  directConnect: true,
+  baseUrl: 'http://localhost:4200/',
+  framework: 'jasmine',
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000,
+    print: function() {}
+  },
+  onPrepare() {
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({
+      spec: {
+        displayStacktrace: StacktraceOption.PRETTY
+      }
+    }));
+  },
+  plugins: [{
+    package: 'protractor-image-comparison',
+    options: {
+      baselineFolder: join(process.cwd(), './screenshotTesting/baseline/'),
```

Comment:
I also see an image file in the `actual/` directory. Is that expected?

Reply:
The image file in the `actual/` directory is the screenshot of the served web page. The image in the `diff/` directory is the difference between the image in the `actual/` directory and the baseline images